articles/mariadb/index.md

```diff
@@ -26,17 +26,17 @@ In this article you will learn how to;
 
 - fetch the saved data from the Database  
 
-### Step 1: Installing Xampp
+## Step 1: Installing Xampp
 
 Xampp comes along with PhpMyAdmin when downloaded into a machine.  PhpMyAdmin is the MariaDatabase that we will be working on within this article.
```

Comment:
Regarding this sentence
`PhpMyAdmin is the MariaDatabase that we will be working on within this article.`
PhpMyAdmin is merely the tool to help manage the database. Make correction